Tests/Recurly/Invoice_Test.php

```diff
@@ -116,6 +117,18 @@ public function testMarkFailed() {
     $invoice->markFailed();
     $this->assertEquals($invoice->state, 'failed');
   }
+  
+  public function testEnterOfflinePayments() {
+    $this->client->addResponse('POST', 'https://api.recurly.com/v2/invoices/1003/transactions', 'invoices/successful_payment-200.xml');
+    $invoice = Recurly_Invoice::get('1003', $this->client);
+    $invoice->enterOfflinePayment(array(
+      'payment_method'=> 'check',
+      'collected_at'=> date('Y/m/d H:i:s'),
+      'amount_in_cents' => 2995,
+      'description' => "Check #12345"
+    ), $this->client);
```

Comment:
I think we need to test the serialization separately. The response matching only compares the URLs, not the request body.